examples/nlp/language_modeling/get_wkt2.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
license header is missing

Reply:
Wait, the file says it is adapted from https://github.com/salesforce/awd-lstm-lm/blob/master/getdata.sh
That repo is not Apache 2.0, it uses BSD-3-Clause License, we shouldn't be putting our license header in it.

Please remove our license header.

Reply:
I removed it. I copied this file from old NeMo.